firebase-document.html

```diff
@@ -73,11 +73,11 @@
      *     not given, a random key will be generated and used.
      * @return {Promise} A promise that resolves once this.data has been
      *     written to the new path.
-     * @override
+     *
      */
     saveValue: function(parentPath, key) {
       return new Promise(function(resolve, reject) {
-        var path;
+        var path = '';
```

Comment:
suggest `null` over `''` since it was possible for `this.path = undefined` before. `null` is closer to `undefined` over `''`